server/lib/notifications.js

```diff
@@ -161,6 +161,17 @@ export async function notifyAdminsOfCollective(CollectiveId, activity, options =
   return notifySubscribers(adminUsers, activity, options);
 }
 
+/**
+ * Notify all admins of hosted collectives
+ */
+export async function notifyHostedCollectiveAdmins(CollectiveId, activity, options = {}) {
+  const collective = await models.Collective.findByPk(CollectiveId);
+  const hostedCollectives = collective.getHostedCollectives();
+
+  hostedCollectives.map(collective => {
+    notifyAdminsOfCollective(collective.id, activity, options);
+  });
```

Comment:
My previous feedback still applies here.

`notifyAdminsOfCollective` triggers at least 2 SQL queries (and usually more) so this loop will trigger more than 2500 SQL queries at the same time for the Open Source Collective.

Ideally we should have one single SQL query that loads all the hosted collective admins emails at once.